airflow/decorators/branch_python.py

```diff
@@ -38,6 +38,8 @@ class _BranchPythonDecoratedOperator(DecoratedOperator, BranchPythonOperator):
         Defaults to False.
     """
 
+    custom_operator_name = "@task.branch"
+
```

Comment:
since https://github.com/apache/airflow/pull/25783 was merged it's not redundant

Reply:
It will be automatically removed on merge :)